src/utils/changeElementVOVisibility.js

```diff
@@ -1,12 +1,44 @@
 /**
  * This source file is part of the Swift.org open source project
  *
- * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Copyright (c) 2022 Apple Inc. and the Swift project authors
  * Licensed under Apache License v2.0 with Runtime Library Exception
  *
  * See https://swift.org/LICENSE.txt for license information
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
+import TabManager from 'docc-render/utils/TabManager';
+
+const OG_PREFIX = 'data-original-';
+const ARIA = 'aria-hidden';
+const TABINDEX = 'tabindex';
+
+function cacheOriginalAttribute(element, prop) {
+  // make sure that prop isn't cached already
+  let originalValue = element.getAttribute(OG_PREFIX + prop);
+  if (!element.hasAttribute(OG_PREFIX + prop) || originalValue === '') {
```

Comment:
Shouldnt we just check  if there is a cached attribute and only cache if there is not?

```suggestion
  if (!element.hasAttribute(OG_PREFIX + prop)) {
```

Reply:
Is there any reason why this function doesn't look like this?

```js
function cacheOriginalAttribute(element, prop) {
  // make sure that prop isn't cached already
  if (element.hasAttribute(OG_PREFIX + prop)) return;

  const originalValue = element.getAttribute(prop) || '';
  element.setAttribute(OG_PREFIX + prop, originalValue);
}
```
I don't understand why adding the originalValue as a let var before.

Reply:
so we wrote the same comment @dobromir-hristov 😆 https://github.com/apple/swift-docc-render/pull/427#discussion_r964889622
actually you wrote it first!

Reply:
Because if the element has a cached attribute, but it's empty; we don't want to return here on this line:
`if (element.hasAttribute(OG_PREFIX + prop)) return;`

We would still want to set the attribute. Did I miss something?

Reply:
then what about `if (!!element.getAttribute(OG_PREFIX + prop)) return;` ?